server/data/mappers/createGoalMapper.test.ts

```diff
@@ -20,6 +20,7 @@ describe('createGoalMapper', () => {
       title: createGoalDto.title,
       category: 'WORK',
       steps: [expectedAddStepRequest1, expectedAddStepRequest2],
+      targetCompletionDate: createGoalDto.targetCompletionDate.toISOString(),
```

Comment:
Rather than put `createGoalDto.targetCompletionDate.toISOString()` in the expected here, we often find its more readable to put the expected value as a literal
```
targetCompletionDate: '2024-01-01'
```
This has a couple of advantages. It makes the test more readable and easier to reason about, and makes it easier to understand what the implementation is actually doing. 👍 

You might also find it highlights a bug with the implementation. You _might_ find (I'm not sure) that the call to `toISOString()` also returns a time component, and we defo don't want that